FiveCalls/FiveCalls/MultipleSelectionSegmentedControl.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  MultipleSelectionSegmentedControl.swift
+//  FiveCalls
+//
+//  Created by Christopher Brandow on 2/7/17.
+//  Copyright © 2017 5calls. All rights reserved.
+//
+
+import Foundation
+
+protocol MultipleSelectionDelegate {
+    func control(changed control: MultipleSelectionControl)
+}
+class MultipleSelectionControl: UIControl {
+
+    //TODO: get values from persistence
+    //TODO: persist values
+    //TODO: override UIControl
```

Comment:
Seems like persistence is being handled by the local notification API?  do we need anything else here?